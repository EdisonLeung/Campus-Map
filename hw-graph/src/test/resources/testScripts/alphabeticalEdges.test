# Test if ListChildren will print the children with edges in alphabetical order
# by adding 3 edges that point to a node in reverse alphabetical order

# Create graph
CreateGraph alphabeticEdge

# Add 2 Nodes
AddNode alphabeticEdge a
AddNode alphabeticEdge b

# Add edges from a to b in reverse alphabetical order
AddEdge alphabeticEdge a b AC
AddEdge alphabeticEdge a b AB
AddEdge alphabeticEdge a b AA

# Print nodes and children
ListNodes alphabeticEdge
ListChildren alphabeticEdge a