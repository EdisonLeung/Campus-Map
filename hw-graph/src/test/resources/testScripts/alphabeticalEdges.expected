# Test if ListChildren will print the children with edges in alphabetical order
# by adding 3 edges that point to a node in reverse alphabetical order

# Create graph
created graph alphabeticEdge

# Add 2 Nodes
added node a to alphabeticEdge
added node b to alphabeticEdge

# Add edges from a to b in reverse alphabetical order
added edge AC from a to b in alphabeticEdge
added edge AB from a to b in alphabeticEdge
added edge AA from a to b in alphabeticEdge

# Print nodes and children
alphabeticEdge contains: a b
the children of a in alphabeticEdge are: b(AA) b(AB) b(AC)